packages/data-context/src/sources/GitDataSource.ts

```diff
@@ -83,8 +83,10 @@ export class GitDataSource {
     // Simple Git will error if the projectRoot does not exist.
     // This should never happen outside of testing code simulating
     // incorrect scenarios
+    debug('config: %o', this.config)
```

Comment:
At my old gig we always removed console logs & debugger statements; at Cypress do we leave debugger statements in?

Reply:
Yes, some of these `debug` statements are way too verbose to put in `cypress:*`. Developers use `DEBUG=cypress:*` frequently to get a high level view of what Cypress is doing, so we should try not to pollute it [more than it already is].

There is actually a `cypress-verbose:*` namespace available in this file as `debugVerbose`, where you can put logs like https://github.com/cypress-io/cypress/blob/383bdb1d3ddcae645dfabc76b65b2aeb319b6216/packages/data-context/src/sources/GitDataSource.ts#L315 and https://github.com/cypress-io/cypress/blob/383bdb1d3ddcae645dfabc76b65b2aeb319b6216/packages/data-context/src/sources/GitDataSource.ts#L234 if you do feel they will be valuable in production.

Reply:
@flotwig I’ll keep this recommendation in mind, but in this specific scenario these statements were there before my code but appear to be under my name in `git blame` because of a revert I needed to do. I think the other debug statements I added (lines 86, 308, and 314) may or may not need to be verbose, what do you think?